style.css

```diff
@@ -773,4 +901,112 @@ textarea {
   .line {
     width: 100%;
   }
+
+  /* -----------------desktop popup----------------- */
+  .modal-mobile {
+    display: none;
+  }
+
+  .modal-desk {
+    overflow-x: hidden;
+    position: fixed;
+    top: 100px;
+    left: 13%;
+    border: 1px solid #ebecf0;
+    box-shadow: 0 48px 48px rgba(0, 0, 0, 0.763);
+    background-color: #fff;
+    width: 75%;
+    height: 80vh;
+    display: none;
+  }
+
+  .modal-des {
+    margin-left: 30px;
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+  }
```

Comment:
- [ ] [REQUIRED] Please correct the differences that I have `highlighted` in the style errors in the image below! Take full advantage of [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)'s debugging time as it has detailed instructions on how to style each element.

![image](https://user-images.githubusercontent.com/63915024/188953968-1bb646ec-bf06-44dd-a8e8-ebe52232cee3.png)